BioArchLinux/vsearch/PKGBUILD

```
@@ -0,0 +1,32 @@
+# Maintainer: Martin Larralde <martin.larralde@embl.de>
+
+pkgname=vsearch
+pkgver=2.25.0
+pkgrel=0
+pkgdesc="Versatile open-source tool for microbiome analysis. https://doi.org/10.7717/peerj.2584"
+arch=('x86_64' 'aarch64' 'ppc64le')
+url="https://github.com/torognes/vsearch"
+license=('BSD-2-Clause OR GPL-3.0-only WITH Autoconf-exception-3.0')
+depends=('glibc' 'zlib' 'bzip2')
+makedepends=('ghostscript')
+checkdepends=()
+source=("https://github.com/torognes/vsearch/archive/refs/tags/v$pkgver.tar.gz")
```

The source filename should be unique according to the [Arch package guidelines](https://wiki.archlinux.org/title/Arch_package_guidelines#Package_sources). Also this can be shortened a bit by using the `$url` variable and removing the `refs/tags` part.
```suggestion
source=("$pkgname-$pkgver.tar.gz::$url/archive/v$pkgver.tar.gz")
```